app/views/admin/promotional_feature_items/_promotional_feature_item.html.erb

```diff
@@ -22,9 +22,5 @@
   </p>
 
   <%= link_to "Edit", edit_admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature, promotional_feature_item), class: "btn btn-default"%>
-  <%= button_to "Delete",
-        admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature, promotional_feature_item),
-        method: "delete",
-        class: "btn btn-danger",
-        data: { confirm: "Are you sure you want to delete this feature item?" } %>
+  <%= link_to "Delete", confirm_destroy_admin_organisation_promotional_feature_item_path(@organisation,@promotional_feature, promotional_feature_item), class: 'btn btn-danger btn-sm' %>
```

Comment:
I think we should continue to use implementation for Bootstrap. i.e. not hit the confirm delete and just use the code deleted here

<img width="698" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/9d9368cc-61c1-4298-8a4c-88e287033716">

We should use link to the confirm delete page for our new implementation though (so on the index page you just added)